cmd/login.go

```diff
@@ -161,6 +155,22 @@ func getProjectContext(ctx *cli.Context, c *cliclient.MasterClient) (string, err
 		return "", nil
 	}
 
+	// If project is given
+	if ctx.String("context") != "" {
```

Comment:
Move this to the top of the function, if user gives `context` let's assume they know there is an existing project to use and not do the list.